CONTRIBUTING.md

```diff
@@ -50,7 +50,37 @@ We use `sphinx` for building documentation. You can call `make build_docs` from
 the docs will be built under `doc/_build/html`.
 
 ## CI
-All PRs triger a Github Actions  build to run linting, type checking, tests, and build docs.
+All PRs triger a Github Actions build to run linting, type checking, tests, and build docs. The status of each 
+Github Action can be viewed on the [actions page](https://github.com/SeldonIO/alibi-detect/actions).
+
+### Debugging via CI
+
+For various reasons, CI runs might occasionally fail. They can often be debugged locally, but sometimes it is helpful
+to debug them in the exact enviroment seen during CI. For this purpose, there is the facilty to ssh directly into
+the CI Guthub Action runner.
+
+#### Instructions
+
+1. Go to the "CI" workflows section on the Alibi Detect GitHub Actions page.
+
+2. Click on "Run Workflow", and select the "Enable tmate debugging" toggle.
+
+3. Select the workflow once it starts, and then select the build of interest (e.g. `ubuntu-latest, 3.10`).
+
+4. Once the workflow reaches the `Setup tmate session` step, click on the toggle to expand it.
+
+5. Copy and paste the `ssh` command that is being printed to your terminal e.g. `ssh TaHAR65KFbjbSdrkwxNz996TL@nyc1.tmate.io`.
+
+6. Run the ssh command locally. Assuming your ssh keys are properly set up for github, you should now be inside the GutHub Action runner.
+
+7. The tmate session is opened after the Python and pip installs are completed, so you should be ready to run `alibi-detect` and debug as required. 
```

Comment:
Not clear to me what you mean here exactly, do you mean `pytest alibi-detect` here?